### ISSUE TITLE
agat_convert_minimap2_bam2gff.pl samtools error

### ISSUE THREAD

**Issue by soungalo** [NBISweden/AGAT#51]
Hello,
I am trying to use agat_convert_minimap2_bam2gff.pl but having some trouble.
I installed AGAT using miniconda and ran minimap2 according to the instructions in the script's help text to produce a sorted bam file. Then I ran the command:
`agat_convert_minimap2_bam2gff.pl -i transcripts_vs_genome.bam > transcripts_vs_genome.gff3`
The run begins and even writes a few lines to the output file, but then crashes with the error:
```
samtools view: writing to standard output failed: Broken pipe
samtools view: error closing standard output: -1
readline() on closed filehandle FILE at /groups/itay_mayrose/liorglic/miniconda3/envs/genomeThreader/bin/agat_convert_minimap2_bam2gff.pl line 224.
```
I've also tried adding the flag -b and using -o instead of redirecting the output, but with the same result. The bam file seems valid - was able to read it with samtools etc.
I am using AGAT v0.4.0, minimap2 v2.17-r941 and samtools v1.10

Any idea what the problem might be?
Thanks!

**Comment by Juke34** [comment]
Maybe something related to the samtools version. Use a Sam file instead like that it skips the step using samtools that converts the bam into Sam.

**Comment by soungalo** [comment]
Thanks for your reply, but I'm still unable to complete the run.
I've tried running the script on a sorted and unsorted SAM (w/o -s). In both cases, I got the error message:
`readline() on closed filehandle FILE at /groups/itay_mayrose/liorglic/miniconda3/envs/genomeThreader/bin/agat_convert_minimap2_bam2gff.pl line 224.`
I also tried using an older version of samtools (0.1.19), but this one seems to be unable to read minimap2 output.
Any other ideas?
